libafl/src/stages/power.rs

```diff
@@ -128,11 +138,16 @@ where
     /// Creates a new transforming [`PowerMutationalStage`]
     pub fn transforming(mutator: M) -> Self {
         Self {
+            name: Cow::Borrowed("PowerMutationalStage"),
             mutator,
             phantom: PhantomData,
             restart_helper: ExecutionCountRestartHelper::default(),
         }
     }
+    /// Sets the name of the `PowerMutationalStage`
+    pub fn set_name(&mut self, name: Cow<'static, str>) {
```

Comment:
I don't think we should have an API to set the name, this might break a lot of assumptions

Reply:
Also, do we have the same constructor twice, here? `new` and `transforming` look identical?

Reply:
You're right. They had different generics but the ``E::Input`` supersedes it I guess